src/main/java/stirling/software/SPDF/SPDFApplication.java

```diff
@@ -26,21 +28,24 @@
 import stirling.software.SPDF.config.ConfigInitializer;
 import stirling.software.SPDF.model.ApplicationProperties;
 
-@SpringBootApplication
-@EnableScheduling
 @Slf4j
-public class SPdfApplication {
+@EnableScheduling
+@SpringBootApplication
+public class SPDFApplication {
+
+    private static final Logger logger = LoggerFactory.getLogger(SPDFApplication.class);
```

Comment:
Remove logger here to maintain the exissting lombok @Slf4j 
Unless there is a reason for this and the renames?

Reply:
I think that came in accidentally while rebasing earlier 😅  will remove